cli_utils/unpackdat.c

```diff
@@ -90,47 +140,105 @@ void close_dat(struct datfile *rf) {
 	return;
 }
 
-#define DCHUNK_SIZE 0x8D	//chunk size is very happy
-#define DCHUNK_DLEN 0x80	//bytes of payload
-// "FMT SRC TGT LEN 34 82 AH AM AL CL <CL bytes of data> <CRC1> <CRC2> <CKS>"
-void unpack_dat(FILE *outf, uint8_t *dest, const uint8_t *src, u32 siz) {
-	// 1- search for "88 34 82"
-	const uint8_t pm[]={0x88, 0x34, 0x82};
+void print_hex(const void *src, int len) {
+	const uint8_t *ptr = src;
+	for (int i = 0; i < len; i++) {
+		printf(" %02x", ptr[i]);
+	}
+	printf("\n");
+}
+
+void unpack_dat(FILE *outf, struct datfile *rf, bool trim) {
```

Comment:
yes, passing in a `struct *` is a better way to do this.
Can you add a comment for argument "trim"